tools/ocean/copernicusmarine.xml

```diff
@@ -2,45 +2,29 @@
     <description>retrieve marine data</description>
     <macros>
         <token name="@TOOL_VERSION@">1.3.3</token>
-        <token name="@VERSION_SUFFIX@">1</token>
+        <token name="@VERSION_SUFFIX@">2</token>
     </macros>
     <requirements>
         <requirement type="package" version="@TOOL_VERSION@">copernicusmarine</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-    #import re
-        #set $cmems_username = $__user__.extra_preferences.get('cmems_account|cmems_username', "")
-        #set $cmems_password = $__user__.extra_preferences.get('cmems_account|cmems_password', "")
-
-        #if $cmems_username == "" or $cmems_password == ""
-            #set $cmems_username = os.getenv('CMEMS_USERNAME', '')
-            #set $cmems_password = os.getenv('CMEMS_PASSWORD', '')
-
-            echo "Using global ENVs" &&
-
-            #if re.match(r"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).{8,}$", $cmems_password):
-                echo "Password meets the required criteria" &&
-            #else:
-                echo "Password does not meet the required criteria" &&
-            #end if
-
-        #end if
-
-        #if $cmems_username == ""
-            echo 'Error. Set your CMEMS credentials via: User -> Preferences -> Manage Information'  &&
-        #end if
+        bash '$cmems_credentials' &&
         
-        #set $filename="config_file_with_my_command"
-        echo -e '$input_text' > $filename --force-download --username $cmems_username --password $cmems_password &&
+        echo -e '$input_text' > config_file_with_my_command &&
+        echo ' --force-download ' >> config_file_with_my_command &&
```

Comment:
ok, this does not work, can you add this force-download as part of your check.py?